src/app/public/modules/text-expand-repeater/text-expand-repeater.component.html

```diff
@@ -1,7 +1,10 @@
 <div class="sky-text-expand-repeater">
   <ul class="sky-text-expand-repeater-container" [id]="contentSectionId" (transitionend)="animationEnd()" #container>
     <li *ngFor="let item of contentItems" class="sky-text-expand-repeater-item">
-      {{item}}
+      <ng-template
```

Comment:
Should this be an `ng-container` ?

Reply:
It could be; `ng-template` is what's being used internally by the `ng-container`, it just adds more functionality that we don't necessarily need, here.